app/pages/incident/EndorsementCountryPage.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.incident
+
+import controllers.incident.routes
+import models.{Index, Mode, UserAnswers}
+import models.reference.Country
+import pages.QuestionPage
+import pages.sections.incident.IncidentSection
+import play.api.libs.json.JsPath
+import play.api.mvc.Call
+
+case class EndorsementCountryPage(index: Index) extends QuestionPage[Country] {
+
+  override def path: JsPath = IncidentSection(index).path \ toString
```

Comment:
Should this not be:
  override def path: JsPath = incident.IncidentSection(index).path \ toString
to keep consistency with EndorsementDatePage and EndoresemenPlacePage ?